stable/burrow/README.md

```diff
@@ -1,164 +1,183 @@
+| `image.repository` | image repository | `"hyperledger/burrow"` |
+| `image.tag` | image tag | `"0.23.1"` |
+| `image.pullPolicy` | image pull policy | `"IfNotPresent"` |
+| `chain.name` | human readable name of the blockchain network | `""` |
+| `chain.numberOfNodes` | number of nodes for the blockchain network | `1` |
+| `chain.logLevel` | log level for the nodes (`debug`, `info`, `warn`) | `"info"` |
+| `chain.extraSeeds` | network seeds to dial in addition to the cluster booted by the chart; each entry in the array should be in the form `ip:port` (note: because P2P connects over tcp, the port is absolutely required) | `[]` |
+| `chain.testing` | - | `false` |
+| `validatorAddresses` | list of validators to deploy | `[]` |
+| `validatorAddresses.Validator_0.Address` | - | `"DEADBEEF"` |
+| `validatorAddresses.Validator_0.NodeAddress` | - | `"DEADBEEF"` |
+| `env` | environment variables to configure burrow | `{}` |
+| `extraArgs` | extra arguments to give to the build in `burrow start` command | `{}` |
+| `organization` | name of the organization running these nodes (used in the peer's moniker) | `"monax"` |
+| `persistence.enabled` | enable pvc for the chain data | `true` |
+| `persistence.size` | size of the chain data pvc | `"80Gi"` |
+| `persistence.storageClass` | storage class for the chain data pvc | `"standard"` |
+| `persistence.accessMode` | access mode for the chain data pvc | `"ReadWriteOnce"` |
+| `persistence.persistentVolumeReclaimPolicy` | does not delete on node restart | `"Retain"` |
+| `peer.service.type` | service type | `"ClusterIP"` |
+| `peer.service.port` | peer port | `26656` |
+| `peer.ingress.enabled` | expose port | `false` |
+| `peer.ingress.hosts` | - | `[]` |
+| `rpcGRPC.enabled` | enable GRPC service | `true` |
+| `rpcGRPC.service.port` | GRPC port | `10997` |
+| `rpcGRPC.service.type` | service type | `"ClusterIP"` |
+| `rpcGRPC.service.loadBalance` | enable load balancing across nodes | `true` |
+| `rpcGRPC.ingress.enabled` | expose port | `false` |
+| `rpcGRPC.ingress.hosts` | - | `[]` |
+| `rpcGRPC.ingress.annotations` | extra annotations | `` |
+| `rpcGRPC.ingress.tls` | - | `` |
+| `rpcInfo.enabled` | enable RPC service | `true` |
+| `rpcInfo.service.port` | RPC port | `26658` |
+| `rpcInfo.service.type` | service type | `"ClusterIP"` |
+| `rpcInfo.service.loadBalance` | enable load balancing across nodes | `true` |
+| `rpcInfo.ingress.enabled` | expose port | `false` |
+| `rpcInfo.ingress.partial` | exposes the `/accounts` and `/blocks` paths externally | `true` |
+| `rpcInfo.ingress.pathLeader` | - | `"/"` |
+| `rpcInfo.ingress.annotations` | extra annotations | `` |
+| `rpcInfo.ingress.hosts` | - | `[]` |
+| `rpcInfo.ingress.tls` | - | `` |
+| `rpcMetrics.enabled` | enable RPC service | `true` |
+| `rpcMetrics.port` | RPC port | `9102` |
+| `rpcMetrics.path` | http endpoint | `"/metrics"` |
+| `rpcMetrics.blockSampleSize` | number of previous blocks to fetch | `100` |
```

Comment:
`number of previous blocks to utilize in calculating the histograms and summaries which are sent to prometheus`